qiskit_experiments/library/characterization/analysis/ramsey_xy_analysis.py

```diff
@@ -108,45 +108,77 @@ def _generate_fit_guesses(
         Returns:
             List of fit options that are passed to the fitter function.
         """
-        y_max = np.max(curve_data.y)
-        y_min = np.min(curve_data.y)
-        y_ptp = y_max - y_min
+        ramx_data = curve_data.get_subset_of("X")
+        ramy_data = curve_data.get_subset_of("Y")
 
         user_opt.bounds.set_if_empty(
-            amp=(0, y_ptp),
+            amp=(0, np.inf),
```

Comment:
Thanks, `2 * ptp` seems good. 
> you would have to have 0 extra oscillation and somehow a phase offset that put the X and Y quadratures away from angle 0 where they get closer to each other

Could you please elaborate?

Reply:
I was considering a very extreme case. For data like this:

![image](https://user-images.githubusercontent.com/1178780/184943115-4f566d6a-0800-4f3f-9a4a-91e836d5b33b.png)

usually we start with the qubit along Y and the two curves at 0.5 and 1, so ptp should be close to amp. If somehow there were a rotation to the qubit so that it was at the position at 0.6 us where the two curves cross and the qubit were not precessing so it just stayed there the ptp would not be close to the amp.

I think it is okay to assume that the user includes 0 delay with 0.5 and 1 points or at least enough data for a good fraction of an oscillation.